tests/bindings/test_statevector_nb.py

```diff
@@ -0,0 +1,364 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for StateVector classes in nanobind-based modules."""
+
+import numpy as np
+import pytest
+from conftest import backend
+
+if backend != "qubit":
+    pytest.skip("Skipping tests for binaries other than lightning_qubit .", allow_module_level=True)
```

Comment:
Is this just meant to be temporary as we implement the bindings in various devices?

Reply:
yes